README.md

```diff
@@ -1,24 +1,221 @@
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+Note: We are assuming you have pre-installed ruby, WSL, ubuntu and rails!
+
+### Prerequisites
+
+In order to run this project you need:
+
+first at all, install rails:
+
+```sh
+gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd your-folder
+  git@github.com:alvp01/hello-rails-react.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd hello-rails-react
+  bundle install
+```
+
+Setting up the Data Base:
+
+```sh
+  "Set up Username and password information in the config/database.yml"
+  rails db:create
+  rails db:migrate
+  rails db:seed
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  ./bin/dev
+```
+
+### Deployment
+
+The deployment will be available soon !
+
+<!--
+Example:
+You can deploy this project using:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Abel Lavieri**
+
+- GitHub: [@cancelei](https://github.com/alvp01)
+- LinkedIn: [Glauber Bannwart](https://www.linkedin.com/in/abel-lavieri)
+
+  
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> We this project can improve, we are planning to include the next features soon:
+
+- [X] **User Interface, Controllers and Models**
+- [N/A] **API documentation**
+- [X] **View Implementation**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/cancelei/hello-rails-react2/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Would you like to help?
+
+If you like this project feel free to leave an start, as well if you have a suggestion feel free to visit issues page.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for creating this exercise-project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Learn more about FAQ below:
+
+- **How do I fix Linters errors if I modify the project?**
+
+  - for rubocop run
+
+  ```sh
+  rubocop -A
+  ```
+
+  - for Stylint run:
+
+  ```sh
+  npx stylelint "**/*.{css,scss}" --fix
+  ```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] Please 🙏🏻  add the `MIT` license properly, your previous code reviewer also suggested you but it seems like you haven't implemented any changes. Including the **MIT** *License* in your `GitHub` repository **README** is crucial for several reasons. Firstly, the **MIT** License is a permissive `open-source` license that grants users the freedom to use, modify, and distribute the software without legal restrictions. By choosing the *MIT* License 🗒️, you are encouraging collaboration, inviting other developers to contribute, and fostering a vibrant open-source community. Additionally, using the *MIT* License helps protect your work by clearly defining the terms and conditions under which it can be used, providing a level of legal 🚨 clarity and ensuring that proper attribution is given. Furthermore, the widespread popularity and familiarity of the *MIT* License make it easy for others to understand and comply with the licensing requirements, which ultimately promotes the adoption and usage of your software 💻 .